docs/getting-started/apple/screenshots.md

```diff
@@ -1,8 +1,258 @@
+- Is your app localized into many languages that require different screenshots for each?
+- Have you made sure that no loading indicators are showing?
+- Is the same content displayed for each of your size variations?
 
+_fastlane_ tools can automate this process making it fast, and consistent while giving you beautiful results!
+
+## Alternatives
+
+For the full story about the many ways that developers can capture and beautify screenshots check out [this article](https://krausefx.com/blog/creating-perfect-app-store-screenshots-of-your-ios-app).
+
+### Manually Capturing Screenshots
+
+It goes without saying that this takes too much time, which also decreases the quality of the screenshots. Since it is not automated, the screenshots will show slightly different content on the various devices and languages. Many companies choose to create screenshots in one language and use them for all languages. While this might seem okay to us developers, there are many potential users out there that cannot read the text on your app screenshots if they are not localised. Have you ever looked at a screenshots with content in a language you don't know? It won't convince you to download the app.
+
+However, the biggest disadvantage of this method is what happens when you need to repeat the process. If you notice a spelling mistake in the screenshots, if you release an update with a new design, or if you just want to show more up to date content, you'll have to create new screenshots for all languages and devices... manually.
+
+The positive side of this approach is that your screenshots will all be crisp and correctly sized, with readable text.
+
+### Scaled Screenshots via iTunes Connect
+
+With the [iTunes Connect screenshot update](https://developer.apple.com/news/?id=08082016a) from August 8th, 2016, you can now use one set of screenshots for all available devices and screenshots. iTunes Connect will automatically scale the images for you, so that each device renders the same exact image.
+
+While this is convenient, this approach has the same problems as the device frame approach: The screenshots don't actually show how the app looks on the user's device. It's a valid way to start though, since you can gradually overwrite screenshots for specific languages and devices.
+
+# Capture Screenshots Automatically
+
+_snapshot_ works with _fastlane_ to automate the process of capturing screenshots of your app. It allows you to:
+
+- Capture hundreds of screenshots in multiple languages on all simulators
+- Do something else while the computer takes the screenshots for you
+- Configure it once, and store the configuration so anyone on the team can run it
+- Generate a beautiful web page showing all screenshots on all devices. This is perfect to send to Q&A, marketing, or translators for verification
+- Avoid having loading indicators in your App Store screenshots by intelligently waiting for network requests to be finished
```

Comment:
We could add

- Get a summary of how your app looks like across all supported devices and languages